### PR TITLE
Added entangler unit tests

### LINKED ISSUE #41: Entangler unit tests

### PR DESCRIPTION
**Pull request by feanz**
resolves #41 

Cleaned up entangler statements by cleaning up newlines tabs and
multispaces

### CODE REVIEW

File: src/lhm.net/Entangler.cs [hunk 1]
```diff
@@ -87,5 +88,13 @@ AFTER DELETE
                             DELETE FROM [{_destination.Name}] WHERE {_destination.PrimaryKey} IN (SELECT {_destination.PrimaryKey} FROM DELETED)
                         END";
         }
+
+        private string Strip(string value)
```
Comment (Amila17) [inline src/lhm.net/Entangler.cs:92]: could be moved into an existing utility class for the future?

File: src/tests/lhm.net.tests.unit/EntanglerTests.cs [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+                    new ColumnInfo {Name = "info"},
+                    new ColumnInfo {Name = "tags"}
+                }),
+                "2015_01_01_01_01_01_111"
+                ),
+                connection.Object);
+            return sut;
+        }
+
+        private static string Strip(string value, bool removeAllSpaces = true)
```
Comment (Amila17) [inline src/tests/lhm.net.tests.unit/EntanglerTests.cs]: If moved to a utility class then code could be re-used.

**Comment by Amila17** [comment]
:shipit:

File: src/lhm.net/Entangler.cs (hunk 1 above)
Comment (feanz) [inline src/lhm.net/Entangler.cs:92]: This one I think I will leave her till it actually used in multiple places.

File: src/tests/lhm.net.tests.unit/EntanglerTests.cs (hunk 2 above)
Comment (feanz) [inline src/tests/lhm.net.tests.unit/EntanglerTests.cs]: Will move into testbase

**Pull request merged**